### PR TITLE
Update changelog 09-March-2022

### PR DESCRIPTION
**Pull request by danimtb**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+### 09-March-2022 - 13:37 CET
+
+- [feature] Merge pull-requests that bump version automatically.
+- [feature] Get requirements from `conan info --dry-build` command.
+- [feature] Use reference with recipe revision for `conan info` commands.
+- [feature] Add possibility to load the configuration from file.
+- [feature] Fix DSL typo for access request job.
```
Comment (danimtb) [inline docs/changelog.md:9]:
```suggestion
- [fix] Fix DSL typo for access request job.
```

**Review by ericLemanissier — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by ericLemanissier** [comment]
is the merging bot still working ? several pull requests should have been merged already:

- https://github.com/conan-io/conan-center-index/pull/9335
- https://github.com/conan-io/conan-center-index/pull/9350
- https://github.com/conan-io/conan-center-index/pull/9467
- https://github.com/conan-io/conan-center-index/pull/9478
- https://github.com/conan-io/conan-center-index/pull/9644
- https://github.com/conan-io/conan-center-index/pull/9652
- https://github.com/conan-io/conan-center-index/pull/9672

**Comment by SSE4** [comment]
> is the merging bot still working ? several pull requests should have been merged already:
> 
> * [at-spi2-atk: bump requirements #9335](https://github.com/conan-io/conan-center-index/pull/9335)
> * [libmysqlclient: fix few issues with all shared + fix FreeBSD + modernize #9350](https://github.com/conan-io/conan-center-index/pull/9350)
> * [libvpx: fix Visual Studio 2022 build + modernize #9467](https://github.com/conan-io/conan-center-index/pull/9467)
> * [add cctag/1.0.1 #9478](https://github.com/conan-io/conan-center-index/pull/9478)
> * [Modernize rtmidi for conan v2 #9644](https://github.com/conan-io/conan-center-index/pull/9644)
> * [xxhash: add version 0.8.1 #9652](https://github.com/conan-io/conan-center-index/pull/9652)
> * [Update changelog 09-March-2022 #9672](https://github.com/conan-io/conan-center-index/pull/9672)

correct, it seems to be broken after the last deployment. thanks for the report. we're investigating, and will keep posting.

**Comment by SSE4** [comment]
> is the merging bot still working ? several pull requests should have been merged already:
> 
> * [at-spi2-atk: bump requirements #9335](https://github.com/conan-io/conan-center-index/pull/9335)
> * [libmysqlclient: fix few issues with all shared + fix FreeBSD + modernize #9350](https://github.com/conan-io/conan-center-index/pull/9350)
> * [libvpx: fix Visual Studio 2022 build + modernize #9467](https://github.com/conan-io/conan-center-index/pull/9467)
> * [add cctag/1.0.1 #9478](https://github.com/conan-io/conan-center-index/pull/9478)
> * [Modernize rtmidi for conan v2 #9644](https://github.com/conan-io/conan-center-index/pull/9644)
> * [xxhash: add version 0.8.1 #9652](https://github.com/conan-io/conan-center-index/pull/9652)
> * [Update changelog 09-March-2022 #9672](https://github.com/conan-io/conan-center-index/pull/9672)

working again, just merged #9335. thanks for your patience!

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**